<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testResult type="org.overture.vdmj.typechecker.tests.external.NewModuleTestCase">
<message column="7" line="11" message="Definition 'n' not used" messageType="warning" number="5000" resource="applyexpr-22.vdm.result"/>
<message column="3" line="34" message="Recursive function has no measure" messageType="warning" number="5012" resource="applyexpr-22.vdm.result"/>
<message column="29" line="47" message="Name 'ApplyExprTest`fxx' is not in scope" messageType="error" number="3182" resource="applyexpr-22.vdm.result"/>
<result/>
</testResult>
